packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -444,12 +442,13 @@ skipInNode(describe)('plugin-meetings', () => {
         testUtils.delayedPromise(bob.meeting.shareScreen()),
         testUtils.waitForEvents([
           {scope: bob.meeting, event: 'meeting:startedSharingLocal'}
-        ]),
+        ], 10000),
         testUtils.waitForEvents([
           {scope: alice.meeting, event: 'meeting:startedSharingRemote'}
         ]),
         testUtils.waitForEvents([{scope: alice.meeting.members, event: 'members:update'}])
           .then((response) => {
+            this.retries(3);
```

Comment:
Should we add the timeout here instead of retries? What happens when the event comes in while the retry is retrying.